src/auth/screens/events.screen.js

```diff
@@ -149,7 +149,7 @@ class Events extends Component {
       case 'MemberEvent':
         return `${userEvent.payload.action}`;
       case 'PublicEvent':
-        return translate('auth.events.issuesEvent', language);
+        return translate('auth.events.publicEvent', language);
```

Comment:
I tried doing this at first, but I don't think it's possible with the current translation implementation. 

From my (new to react-native world) understanding, the repository link is prepended to the sentence around here: https://github.com/gitpoint/git-point/blob/master/src/auth/screens/events.screen.js#L492 and the payload is empty in this action.

Splitting the sentence in two parts and using getConnector() to inject "public" seemed quiet hackish to me, so I settled on a poorer phrasing 😊 

I'd appreciate it if one of you guys could have a look and fix my PR the proper way so that I could learn from it.